.github/workflows/test.yml

```diff
@@ -41,3 +41,10 @@ jobs:
       run: |
         go version
         go test ./...
+  test-conventions:
```

Comment:
I'd call this `code-checks` or similar, since we can make it do much more later.